### DISCUSSION TITLE
Default install copies enet headers into prefix

### DISCUSSION THREAD

**Discussion by parkerlreed** [LizardByte/community#293] [Sunshine] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Running the default 

```
cmake --install
```

Copies enet into the prefix, thus causing a later conflict if someone tries to install enet on its own.

```
error: failed to commit transaction (conflicting files)
enet: /usr/include/enet/callbacks.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/enet.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/list.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/protocol.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/time.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/types.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/unix.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/utility.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/win32.h exists in filesystem (owned by sunshine-git)
Errors occurred, no packages were upgraded.
```

### Expected Behavior

enet sources to not be included as they are not needed for sunshine operation

### Host Operating System

Linux

### Operating System Version

Arch Linux

### Architecture

64 bit

### Sunshine commit or version

f1452d2

### Package

Linux - AUR (Third Party)

### GPU Type

AMD

### GPU Model

AMD ATI 0a:00.0 Phoenix1

### GPU Driver/Mesa Version

mesa 23.1.6

### Config

```Shell
Empty N/A
```

### Relevant log output

```shell
error: failed to commit transaction (conflicting files)
enet: /usr/include/enet/callbacks.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/enet.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/list.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/protocol.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/time.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/types.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/unix.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/utility.h exists in filesystem (owned by sunshine-git)
enet: /usr/include/enet/win32.h exists in filesystem (owned by sunshine-git)
Errors occurred, no packages were upgraded.
```

**Comment by ReenigneArcher** [comment]
We don't support this `sunshine-git` AUR package. The only one we support is `sunshine`.

Reviewing their `PKGBUILD`, I would personally avoid using it. It's overly complicated and 

If you want nightly builds, we publish our own PKGBUILD and prebuilt package to our nightly release.

https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/installation.html#archlinux-pkgbuild

https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/installation.html#archlinux-pkg

adjust urls as needed: https://github.com/LizardByte/Sunshine/releases/tag/nightly-dev

**Reply by parkerlreed** [reply]
Thanks!

**Reply by Miyuutsu** [reply]
This is definitely still an issue when using the PKGBUILD from https://github.com/LizardByte/Sunshine/releases/tag/nightly-dev.

**Comment by parkerlreed** [comment]
Wait this is still a sunshine issue as I expected. I just used the PKGBUILD from nightly and also checked the prebuilt package. enet headers are still included when they shouldn't be.

```
[parker@rogally sun]$ pacman -Ql sunshine | grep include
sunshine /usr/include/
sunshine /usr/include/enet/
sunshine /usr/include/enet/callbacks.h
sunshine /usr/include/enet/enet.h
sunshine /usr/include/enet/list.h
sunshine /usr/include/enet/protocol.h
sunshine /usr/include/enet/time.h
sunshine /usr/include/enet/types.h
sunshine /usr/include/enet/unix.h
sunshine /usr/include/enet/utility.h
sunshine /usr/include/enet/win32.h
```

![image](https://github.com/LizardByte/Sunshine/assets/841440/95179edb-c59a-40a0-91fa-91df9bfe7c77)

**Comment by ReenigneArcher** [comment, marked as answer]
This was recently reported on Windows, so I decided to look into a little closer. The cause is from a change in the `moonlight-common-c` library and ultimately from a change in the `enet` library, which is a submodule of `moonlight-common-c`.

This is the commit causing it. https://github.com/cgutman/enet/commit/9fda19e54b01919b07d70c944c42576c4f1ef0b9

[a comment by an automated account is not shown]
